src/templates/landing_a.js

```diff
@@ -21,10 +21,22 @@ const Landing = (props) => {
   const [inLocation, setInLocation] = React.useState("");
 
   const applySchollarship = data.allLandingYaml.edges[0].node?.apply_schollarship
-  const programs = data.allChooseProgramYaml.edges[0].node.programs.map(p => ({
+  const landing_utm_course = yml.meta_info.utm_course
+
+  const filteredPrograms  = data.allChooseProgramYaml.edges[0].node.programs.filter((course_el) => {
```

Comment:
here it begins to filter `allChooseProgramYaml` according to the `utm_course` specified in the array